api-gateway-config/scripts/lua/management/apis.lua

```diff
@@ -287,7 +287,11 @@ function getAllAPIs(queryParams)
         apiList[#apiList+1] = cjson.decode(v)
       end
     end
-    apiList = cjson.encode(apiList)
+    if next(apiList) == nil then
+      apiList = "[]"
```

Comment:
hmm interesting approach. @mdeuser would this work for your OpenWhisk test cases?